config/application.rb

```diff
@@ -43,6 +43,7 @@ class Application < Rails::Application
     config.eager_load_paths << Rails.root.join('enterprise/listeners')
     # rubocop:disable Rails/FilePath
     config.eager_load_paths += Dir["#{Rails.root}/enterprise/app/**"]
+    config.eager_load_paths << Rails.root.join('enterprise/app/models/concerns')
```

Comment:
why do we need this ? won't the previous line will load the files into the load paths ?

Reply:
It doesn't, it just lists the files in the folders in that directory alone, doesn't go a step further down the tree. I tried adding `**/**` but that didn't work for some reason